examples/changelog.rs

```diff
@@ -20,13 +20,21 @@ use reql::{Client, Document, Run};
  *
  * setup a "test" database, with a "test" table, and run:
  *
- * r.db('test').table('test').insert({ test: 1 })
+ * // Insert an item
+ * r.db('test').table('test').insert({ test: 1 });
```

Comment:
Rust doesn't support anonymous documents like `{ test: 1 }` use the [json macro](https://docs.rs/serde_json/1.0.5/serde_json/macro.json.html) instead. Also note that single quotes are used for `char`. So this query should be:-

```rust
r.db("test").table("test").insert(json!({ "test": 1 }));
```

Reply:
These actions are actually javascript, as they were intended to be input in the rethinkdb admin panel's data-explorer.

If you think it's more useful, I can add them at the end of the file as actual rust commands, which execute when the file is run?

Reply:
Oop! My bad. Never mind, then. I had not read your instructions just before those commands.

Reply:
> Edit: Perhaps we can add the equivalent commands below, so as to show how to run these inserts in another thread, I'll do that just now.

That's not really necessary but if you are up for it, then knock yourself out :)